extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -115,28 +115,70 @@ function render_player( $player_data, $attributes ) {
 		$player_data
 	);
 
+	// Set CSS classes for `secondary` and `background` colors.
+	$secondary_color         = isset( $attributes['secondaryColor'] ) ? $attributes['secondaryColor'] : null;
+	$custom_secondary_color  = isset( $attributes['customSecondaryColor'] ) ? $attributes['customSecondaryColor'] : null;
+	$background_color        = isset( $attributes['backgroundColor'] ) ? $attributes['backgroundColor'] : null;
+	$custom_background_color = isset( $attributes['customBackgroundColor'] ) ? $attributes['customBackgroundColor'] : null;
+
+	$secondary_color_class  = get_color_class_name( 'color', $secondary_color );
+	$background_color_class = get_color_class_name( 'background-color', $background_color );
+
+	$secondary_classes_name = '';
+	$secondary_inline_style = '';
+	if ( isset( $secondary_color_class ) || isset( $custom_secondary_color ) ) {
+		$secondary_classes_name .= ' has-secondary';
+		if ( isset( $secondary_color_class ) ) {
+			$secondary_classes_name .= $secondary_color_class;
+		} elseif ( isset( $custom_secondary_color ) ) {
+			$secondary_inline_style .= "color: $custom_secondary_color;";
+		}
+	}
+
+	$background_classes_name = '';
+	$background_inline_style = '';
+	if ( isset( $background_color_class ) || isset( $custom_background_color ) ) {
+		$background_classes_name .= ' has-background';
+		if ( isset( $background_color_class ) ) {
+			$background_classes_name .= $background_color_class;
+		} elseif ( isset( $custom_background_color ) ) {
+			$background_inline_style .= "background-color: $custom_background_color;";
+		}
+	}
+
+	$podcast_player_classes_name = trim( $secondary_classes_name . $background_classes_name );
+	$podcast_player_inline_style = trim( $secondary_inline_style . ' ' . $background_inline_style );
+
 	$block_classname = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attributes, array( 'is-default' ) );
 	$is_amp          = ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() );
 
 	ob_start();
 	?>
 	<div class="<?php echo esc_attr( $block_classname ); ?>" id="<?php echo esc_attr( $instance_id ); ?>">
-		<ol class="jetpack-podcast-player__episodes">
-			<?php foreach ( $player_data['tracks'] as $attachment ) : ?>
-			<li class="jetpack-podcast-player__episode">
-				<a
-					class="jetpack-podcast-player__episode-link"
-					href="<?php echo esc_url( $attachment['link'] ); ?>"
-					role="button"
-					aria-pressed="false"
+		<section
```

Comment:
Do we need that additional level with `section`, or could we roll this in the parent `div` instead?

Reply:
What I' trying to do here is rendering the HTML code as similar as possible [the client does](https://github.com/Automattic/jetpack/blob/master/extensions/blocks/podcast-player/components/podcast-player.js#L176).

Reply:
so the answer is `yes` :-D